gemma-web/src/main/webapp/scripts/api/entities/GemmaNavigationHeader.js

```diff
@@ -32,10 +40,9 @@ Gemma.GemmaNavigationHeader = Ext
          showAbout : function() {
             var w = new Ext.Window(
                {
-                  width : 500,
-                  height : 300,
+                  width : 800,
+                  height : 700,
                   title : "About Gemma",
-                  layout : 'fit',

```

Comment:
There should be a layout that work well for fitting the second panel in the remaining space.